README.md

```diff
@@ -6,7 +6,7 @@ registration code and use it in Beeper Mini.
 ## Supported MacOS versions
 The tool is currently quite hacky, so it only works on specific versions of macOS.
 
-* Intel: 11.5 - 11.7, 13.3.1, 13.5 - 13.6, 14.0 - 14.3
+* Intel: 10.14.6, 10.15.1 - 10.15.7, 11.5 - 11.7, 13.3.1, 13.5 - 13.6, 14.0 - 14.3
```

Comment:
Since the High Sierra offsets were added in [`e018389` (#12)](https://github.com/beeper/mac-registration-provider/pull/12/commits/e01838920a4c5adacd813c22dd53e113d269e95b), this should be updated:

```suggestion
* Intel: 10.13.6, 10.14.6, 10.15.1 - 10.15.7, 11.5 - 11.7, 13.3.1, 13.5 - 13.6, 14.0 - 14.3
```

Reply:
I have not confirmed if Go binaries built by a `macOS 11` runner will run on `macOS 10.13.6`. 

The `macOS 13` runner is currently used by default and `macOS 10.14.x` is unable to run the Go binary so my other PR lowers the runner to `macOS 11` which does allow `macOS 10.14.x` to run the built binary... but someone on `macOS 10.13.6` would need to test if the Go binary built on `macOS 11` also runs on `macOS 10.13.6`. 

For this reason, I was unsure if listing `macOS 10.13.6` prior to confirmation was ideal. Yes, the project would run on `macOS 10.13.6` if built locally on a machine running said version but official "Releases" in the repo may or may not.  

I have no issues adding `macOS 10.13.6` to README via this current PR if others think it should be added.

Reply:
Fair enough, that makes sense. My thought was mostly because you added the offsets for `10.13.6`, so the tool would work on a binary from that version of macOS (eg. if running on another machine)

As for the specifics you mention above, I'll delegate to letting someone 'official' answer that; this was just my personal /2c.

Reply:
I should also note, I can also post offsets for 10.12, 10.11 etc.. but I stopped at 10.13.6 (last High Sierra update) due to the macOS11 runner generated binaries likely not working below 10.13.6 (possibly not even 10.13.6).

If members officially tied to the project wouldn't mind us adding offsets the auto-builds can't support, requiring users to build themselves, I will add the additional offsets. The other PRs I've made to add support for 10.15 and below should with 10.12, 10.11, 10.10, 10.9, 10.8, 10.7... and possibly lower. Though we then might run into issues with the OS not supporting Go or the current Go code properly. 

So... how low can we go? :)

Reply:
Curious, how do you have access to the older binaries? Extracting them from Time Machine backups/similar, or?

Reply:
Downloading macOS and extracting the binary for each version. 

https://github.com/corpnewt/gibMacOS
https://support.apple.com/kb/DL2052
etc

Reply:
Oh cool! Didn't know about that :)

Reply:
> I should also note, I can also post offsets for 10.12, 10.11 etc.. but I stopped at 10.13.6 (last High Sierra update) due to the macOS11 runner generated binaries likely not working below 10.13.6 (possibly not even 10.13.6)

@jetfir3 It's sort of OOS of this PR, but I noticed that there are currently hashes for `12.6.3` / `13.2.1`, but seemingly no offsets for them. Are those versions you're able to access with your method above? ([Ref](https://github.com/beeper/mac-registration-provider/pull/12#discussion_r1440476750))

Also, a few other issues that seem to be seeking different versions, if you were interested:

- https://github.com/beeper/mac-registration-provider/issues/10
  - > `11.0.1/20B29/amd64 (hash: 1a9cc9305e1621a2024a0f2c70fe77af0d0cf1fbd7e971ecd443f2fd587b12fc)`
- https://github.com/beeper/mac-registration-provider/issues/17
  - > `13.6.1/22G313/amd64 (hash: fff8db27fef2a2b874f7bc6fb303a98e3e3b8aceb8dd4c5bfa2bad7b76ea438a)`
- https://github.com/beeper/mac-registration-provider/issues/6
  - > `14.1 Beta (23B5056e)`